UnoCheck/Checkups/WindowsRegistryCheckup.cs

```diff
@@ -0,0 +1,40 @@
+﻿#nullable enable
+
+using DotNetCheck.Models;
+using DotNetCheck.Solutions;
+using Microsoft.Win32;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Checkups
+{
+	public class WindowsRegistryCheckup : Checkup
```

Comment:
Let's make it explicitly about checking the long path support:

```suggestion
	public class WindowsLongPathCheckup : Checkup
```

Reply:
Done!